[2021-11-10 16:19:12] [WARNING]: 登录需要滑条验证码, 请使用手机QQ扫描二维码以继续登录. 
[2021-11-10 16:19:12] [ERROR]: Protocol -> unexpected disconnect: read tcp 192.168.31.25:57012->113.96.12.85:8080: use of closed network connection 
[2021-11-10 16:19:34] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:19:34] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":67,"packet_sent":61,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636532372}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:19:39] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:19:39] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":67,"packet_sent":61,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636532377}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:19:42] [ERROR]: Protocol -> register client failed: Packet timed out 
[2021-11-10 16:19:44] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:19:44] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":67,"packet_sent":61,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636532382}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:19:49] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:19:49] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":67,"packet_sent":61,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636532387}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:19:54] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:19:54] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":67,"packet_sent":61,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636532392}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:19:59] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:19:59] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":68,"packet_sent":62,"packet_lost":0,"message_received":0,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":0}},"time":1636532397}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:02] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:02] [WARNING]: 上报Event数据 {"anonymous":null,"font":0,"group_id":913941037,"message":"[CQ:image,file=13bb67fc3d90971f1e9472b5aac5cd26.image,url=https://gchat.qpic.cn/gchatpic_new/2237553939/913941037-2292750263-13BB67FC3D90971F1E9472B5AAC5CD26/0?term=3,subType=0]","message_id":1438806638,"message_seq":1323653,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=13bb67fc3d90971f1e9472b5aac5cd26.image,subType=0]","self_id":501273515,"sender":{"age":0,"area":"","card":"真正的萌新","level":"","nickname":"天","role":"admin","sex":"unknown","title":"胖虎","user_id":2237553939},"sub_type":"normal","time":1636532400,"user_id":2237553939}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:03] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:03] [WARNING]: 上报Event数据 {"anonymous":null,"font":0,"group_id":913941037,"message":"[CQ:image,file=bac3cbe4bc1be0e3f0bb3b1bdea86585.image,url=https://gchat.qpic.cn/gchatpic_new/2237553939/913941037-2506766898-BAC3CBE4BC1BE0E3F0BB3B1BDEA86585/0?term=3,subType=0]","message_id":-878254131,"message_seq":1323654,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=bac3cbe4bc1be0e3f0bb3b1bdea86585.image,subType=0]","self_id":501273515,"sender":{"age":0,"area":"","card":"真正的萌新","level":"","nickname":"天","role":"admin","sex":"unknown","title":"胖虎","user_id":2237553939},"sub_type":"normal","time":1636532401,"user_id":2237553939}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:04] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:04] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":70,"packet_sent":62,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532402}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:09] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:09] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":70,"packet_sent":62,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532407}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:14] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:14] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":70,"packet_sent":62,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532412}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:19] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:19] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":70,"packet_sent":62,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532417}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:24] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:24] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":70,"packet_sent":62,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532422}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:29] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:29] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":71,"packet_sent":63,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532427}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:34] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:34] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":71,"packet_sent":63,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532432}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:39] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:39] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":71,"packet_sent":63,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532437}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:44] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:44] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":71,"packet_sent":63,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532442}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:49] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:49] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":71,"packet_sent":63,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532447}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:54] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:54] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":71,"packet_sent":63,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532452}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:20:59] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:20:59] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":72,"packet_sent":64,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532457}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:21:04] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:21:12] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":72,"packet_sent":64,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532462}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:21:14] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:21:14] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":72,"packet_sent":64,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532472}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:21:19] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:21:19] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":72,"packet_sent":64,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532477}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:21:24] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:21:24] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":72,"packet_sent":64,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532482}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:21:29] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:21:29] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":73,"packet_sent":65,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532487}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:21:34] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:21:34] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":73,"packet_sent":65,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532492}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:21:39] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:21:39] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":73,"packet_sent":65,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532497}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:21:44] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:21:44] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":73,"packet_sent":65,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532502}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:21:49] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:21:49] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":73,"packet_sent":65,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532507}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:21:54] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:21:54] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":73,"packet_sent":65,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532512}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:21:59] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:21:59] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":74,"packet_sent":66,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532517}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:04] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:04] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":74,"packet_sent":66,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532522}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:09] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:09] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":74,"packet_sent":66,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532527}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:14] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:14] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":74,"packet_sent":66,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532532}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:19] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:19] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":74,"packet_sent":66,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532537}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:24] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:24] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":74,"packet_sent":66,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532542}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:29] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:29] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":75,"packet_sent":67,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532547}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:34] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:34] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":75,"packet_sent":67,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532552}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:39] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:39] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":75,"packet_sent":67,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532557}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:44] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:44] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":75,"packet_sent":67,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532562}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:49] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:49] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":75,"packet_sent":67,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532567}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:54] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:54] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":75,"packet_sent":67,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532572}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:22:59] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:22:59] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":77,"packet_sent":69,"packet_lost":0,"message_received":2,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532401}},"time":1636532577}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:03] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:03] [WARNING]: 上报Event数据 {"anonymous":null,"font":0,"group_id":913941037,"message":"金克丝大招不太好","message_id":-1130244261,"message_seq":1323655,"message_type":"group","post_type":"message","raw_message":"金克丝大招不太好","self_id":501273515,"sender":{"age":0,"area":"","card":"裁诗为骨玉为神","level":"","nickname":"裁诗为骨玉为神","role":"member","sex":"unknown","title":"","user_id":2783939233},"sub_type":"normal","time":1636532580,"user_id":2783939233}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:04] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:04] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":78,"packet_sent":69,"packet_lost":0,"message_received":3,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532581}},"time":1636532582}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:08] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:08] [WARNING]: 上报Event数据 {"anonymous":null,"font":0,"group_id":1021160530,"message":"[CQ:image,file=794b343eaabc4d886c9fe02271814fa4.image,url=https://gchat.qpic.cn/gchatpic_new/38866200/1021160530-2361950403-794B343EAABC4D886C9FE02271814FA4/0?term=3,subType=0]\r\nYAZIMIQI凡士林果酸身体乳烟酰胺保湿滋润秋冬持久留香男女润肤\r\n【在售价】49.9元\r\n【券后价】9.90元\r\n-----------------\r\n【下单方法】领券下单链接： https://s.click.taobao.com/bzq3Geu\r\n淘口令；￥G7yuXwdegCv￥","message_id":-818657162,"message_seq":24300,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=794b343eaabc4d886c9fe02271814fa4.image,subType=0]\r\nYAZIMIQI凡士林果酸身体乳烟酰胺保湿滋润秋冬持久留香男女润肤\r\n【在售价】49.9元\r\n【券后价】9.90元\r\n-----------------\r\n【下单方法】领券下单链接： https://s.click.taobao.com/bzq3Geu\r\n淘口令；￥G7yuXwdegCv￥","self_id":501273515,"sender":{"age":0,"area":"","card":"","level":"","nickname":"颜夕大表哥","role":"admin","sex":"unknown","title":"","user_id":38866200},"sub_type":"normal","time":1636532586,"user_id":38866200}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:09] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:09] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":79,"packet_sent":69,"packet_lost":0,"message_received":4,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532586}},"time":1636532587}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:14] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:14] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":80,"packet_sent":70,"packet_lost":0,"message_received":4,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532586}},"time":1636532592}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:19] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:19] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":81,"packet_sent":71,"packet_lost":0,"message_received":4,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532586}},"time":1636532597}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:24] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:24] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":81,"packet_sent":71,"packet_lost":0,"message_received":4,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532586}},"time":1636532602}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:27] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:27] [WARNING]: 上报Event数据 {"anonymous":null,"font":0,"group_id":913941037,"message":"伤害还行，炸完那一下，就容易暴毙了","message_id":631801569,"message_seq":1323656,"message_type":"group","post_type":"message","raw_message":"伤害还行，炸完那一下，就容易暴毙了","self_id":501273515,"sender":{"age":0,"area":"","card":"裁诗为骨玉为神","level":"","nickname":"裁诗为骨玉为神","role":"member","sex":"unknown","title":"","user_id":2783939233},"sub_type":"normal","time":1636532604,"user_id":2783939233}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:29] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:29] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":83,"packet_sent":72,"packet_lost":0,"message_received":5,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532605}},"time":1636532607}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:34] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:34] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":83,"packet_sent":72,"packet_lost":0,"message_received":5,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532605}},"time":1636532612}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:39] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:39] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":83,"packet_sent":72,"packet_lost":0,"message_received":5,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532605}},"time":1636532617}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:41] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:41] [WARNING]: 上报Event数据 {"client":{"app_id":537097074,"device_kind":"Android","device_name":"Android"},"notice_type":"client_status","online":false,"post_type":"notice","self_id":501273515,"time":1636532619}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:44] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:44] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":86,"packet_sent":72,"packet_lost":0,"message_received":5,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532605}},"time":1636532622}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:49] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:49] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":86,"packet_sent":72,"packet_lost":0,"message_received":5,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532605}},"time":1636532627}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:54] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:54] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":86,"packet_sent":72,"packet_lost":0,"message_received":5,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532605}},"time":1636532632}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:23:59] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:23:59] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":87,"packet_sent":73,"packet_lost":0,"message_received":5,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532605}},"time":1636532637}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:04] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:04] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":87,"packet_sent":73,"packet_lost":0,"message_received":5,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532605}},"time":1636532642}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:09] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:09] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":87,"packet_sent":73,"packet_lost":0,"message_received":5,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532605}},"time":1636532647}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:10] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:10] [WARNING]: 上报Event数据 {"anonymous":null,"font":0,"group_id":913941037,"message":"[CQ:image,file=a95ff1a8f8a8ef5e368d0cde4d81c2d9.image,url=https://gchat.qpic.cn/gchatpic_new/2278694204/913941037-2944785967-A95FF1A8F8A8EF5E368D0CDE4D81C2D9/0?term=3,subType=0]","message_id":1387247223,"message_seq":1323657,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=a95ff1a8f8a8ef5e368d0cde4d81c2d9.image,subType=0]","self_id":501273515,"sender":{"age":0,"area":"","card":"","level":"","nickname":"Decoy","role":"member","sex":"unknown","title":"","user_id":2278694204},"sub_type":"normal","time":1636532647,"user_id":2278694204}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:11] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:11] [WARNING]: 上报Event数据 {"anonymous":null,"font":0,"group_id":913941037,"message":"[CQ:image,file=b4116e568fa40ecc608e44aefc736c2d.image,url=https://gchat.qpic.cn/gchatpic_new/2278694204/913941037-2924782109-B4116E568FA40ECC608E44AEFC736C2D/0?term=3,subType=0]","message_id":-1039094810,"message_seq":1323658,"message_type":"group","post_type":"message","raw_message":"[CQ:image,file=b4116e568fa40ecc608e44aefc736c2d.image,subType=0]","self_id":501273515,"sender":{"age":0,"area":"","card":"","level":"","nickname":"Decoy","role":"member","sex":"unknown","title":"","user_id":2278694204},"sub_type":"normal","time":1636532648,"user_id":2278694204}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:14] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:14] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":89,"packet_sent":73,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532652}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:19] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:19] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":89,"packet_sent":73,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532657}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:24] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:24] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":89,"packet_sent":73,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532662}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:29] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:29] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":90,"packet_sent":74,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532667}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:34] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:34] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":90,"packet_sent":74,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532672}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:39] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:39] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":90,"packet_sent":74,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532677}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:44] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:44] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":90,"packet_sent":74,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532682}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:49] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:49] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":90,"packet_sent":74,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532687}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:54] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:54] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":90,"packet_sent":74,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532692}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:24:59] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:24:59] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":91,"packet_sent":75,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532697}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:25:04] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:25:04] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":91,"packet_sent":75,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532702}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:25:09] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:25:09] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":91,"packet_sent":75,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532707}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:25:14] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:25:14] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":91,"packet_sent":75,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532712}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:25:19] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:25:19] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":91,"packet_sent":75,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532717}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:25:24] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:25:24] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":91,"packet_sent":75,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532722}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:25:29] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
[2021-11-10 16:25:29] [WARNING]: 上报Event数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":501273515,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"packet_received":92,"packet_sent":76,"packet_lost":0,"message_received":7,"message_sent":0,"disconnect_times":1,"lost_times":0,"last_message_time":1636532649}},"time":1636532727}
 到 127.0.0.1:5701 失败: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 
[2021-11-10 16:25:34] [WARNING]: 上报Event到 HTTP 服务器 127.0.0.1:5701 时出现错误: Post "http://127.0.0.1:5701": dial tcp 127.0.0.1:5701: connectex: No connection could be made because the target machine actively refused it. 将重试. 
